package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Isaac Z. Schlueter <i@izs.me> (http://blog.izs.me/)",
   "name": "npmlog",
   "description": "logger for npm",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
We bump versions as a part of the release process. Please don't bump them in your PR.